Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs

```diff
@@ -183,7 +187,6 @@ internal static bool ReconcileOculusIntegrationDefine()
         /// the MRTK source is from the repo, warnings are converted to errors. Warnings are not converted to errors if the MRTK source is from the unity packages.
         /// Warning 618 and 649 are logged when Oculus Integration is imported into the project, 618 is the obsolete warning and 649 is a null on start warning.
         /// </summary>
-        [MenuItem("Mixed Reality Toolkit/Utilities/Oculus/Configure CSC File for Oculus")]
         static void UpdateCSC()
```

Comment:
Does this handle the case where this is being consumed outside the repo and no csc file is present?

Reply:
I'm not sure since the initial code was derived from the leap motion process, I can double check to make sure that nothing crashes/make the appropriate adjustments